### ISSUE TITLE
make_test_graphs.py fails to open file in binary mode, breaks in py3

### ISSUE THREAD

**Issue by nharada1** [tensorflow/tensorflow#8612]
The file //tensorflow/compiler/aot/tests/make_test_graphs.py attempts to write binary data to files without opening the files in binary mode, which raises errors in Python 3. This can be fixed by opening the files in binary mode, i.e. 

```
74     with open(saver_file, 'wb') as f:
75         f.write(saver.as_saver_def().SerializeToString())
```

I'm not sure if making this change will break Python 2. I will try and submit a pull request later in the week.

**Comment by drpngx** [comment]
Thanks, that sounds like a good PR to send!

**Comment by drpngx** [comment]
Please mention this bug in the PR (`Fixes #8612` will make it auto-close this issue).
